### PR TITLE
CED-233: GPU Checkpointing cleanup 

### PR DESCRIPTION
**Pull request by nravic**
Remove env var and trigger gpu checkpoints over rpc.

### CODE REVIEW

File: cmd/daemon.go [hunk 1]
```diff
@@ -49,15 +42,15 @@ var startDaemonCmd = &cobra.Command{
 		if viper.GetBool("profiling_enabled") {
 			go startProfiler()
 		}
-
-		if viper.GetBool("gpu_enabled") {
-			err := pullGPUBinary(ctx, gpuControllerBinaryName, gpuControllerBinaryPath)
+		gpuEnabled, _ := cmd.Flags().GetBool(gpuEnabledFlag)
```
Comment (nravic) [inline cmd/daemon.go:45]: add flag to specify cuda path, we determine version

File: api/restore.go [hunk 2]
```diff
@@ -516,6 +516,7 @@ func (s *service) restore(ctx context.Context, args *task.RestoreArgs) (*int32,
 	}
 
 	var gpuCmd *exec.Cmd
+	// No GPU flag passed in args - if state.GPUCheckpointed = true, always restore using gpu-controller
```
Comment (yashanand1910) [inline api/restore.go:519]: Nice!

File: utils/config.go [hunk 3]
```diff
@@ -17,6 +17,13 @@ const (
 	configFilePerm = 0644
 )
 
+const (
```
Comment (yashanand1910) [inline utils/config.go:20]: I'm guessing this will be updated in a later PR?

**Review by yashanand1910 — APPROVED** [review]
Lgtm!

File: utils/config.go (hunk 3 above)
Comment (nravic) [reply utils/config.go:20]: what needs to be updated?

Comment (yashanand1910) [reply utils/config.go:20]: Sorry, I meant the names of the GPU binaries/libs.

Comment (nravic) [reply utils/config.go:20]: oh yeah we can change stuff around once we start pulling from cloudsmith

**Pull request merged**